res/database/Default/user/sjaele.xml

```diff
@@ -1068,11 +1068,11 @@
 		<news id="Sjaele-news-Koriatis-Zusammenbruch" type="0" creator="7430" created_by="Sjaele">
 			<title>
 				<de>K. Koriatis Zusammenbruch</de>
-
+				<en>K. Koriati's collapse</en>
 			</title>
 			<description>
 				<de>Costa brach beim Bierfest in Chemnitz zusammen. Inzwischen ist er wieder fit.</de>
-
+				<en>Costa collapsed at the beer festival in Chemnitz. He has since recovered.</en>
```

Comment:
Fix name in title ("C.")?

Reply:
Probably because I wasn't sure if I got the reference. https://de.wikipedia.org/wiki/Costa_Cordalis ?